setup.cfg

```
@@ -71,6 +71,7 @@ doc =
     sphinx-bootstrap-theme
     sphinx-rtd-theme
 full =
+    Firefly-vis>=2.0.0
```

@agurvich could you use lower case here and sort in in alpha order please ?
Sorry for the late nitpick
```suggestion
    firefly-vis>=2.0.0
```